Update docs for Digest (#12889)
ydb-platform/ydb: ydb/docs/en/core/yql/reference/yql-core/udf/list/digest.md

* `Digest::XXH3(String{Flags:AutoMap}) -> Uint64`
* `Digest::XXH3_128(String{Flags:AutoMap}) -> Tuple<Uint64,Uint64>`

> The functions for the hashes which support init/seed parameter, accept its value in the optional named argument `Init`.

Reviewer: ```suggestion
The functions for the hashes that support the initialization parameter (seed) accept its value in the optional named argument named `Init`.
```
